lms/djangoapps/courseware/views/views.py

```diff
@@ -994,6 +1015,10 @@ def course_about(request, course_id):
             # context. This value is therefor explicitly set to render the appropriate header.
             'disable_courseware_header': True,
             'pre_requisite_courses': pre_requisite_courses,
+            'course_requirements': course_requirements if course_requirements else None,
```

Comment:
You can handle this in Pearson Core when calling `run_extension_point` for _PEARSON_CORE_STUDENT_COURSE_REQUIREMENTS_FOR_COURSEWARE_. @anfbermudezme